pkg/audit/reporter.go

```diff
@@ -66,7 +66,7 @@ func (reporter *Reporter) RecordAudits(ctx context.Context, req *Report) (failed
 	var errlist errs.Group
 
 	retries := 0
-	for retries < reporter.maxRetries {
+	for retries <= reporter.maxRetries {
 		if len(successes) == 0 && len(fails) == 0 && len(offlines) == 0 && len(pendingAudits) == 0 {
```

Comment:
Maybe it would be clearer by renaming "retries" to "tries", because as it is, when I will have forgotten this PR and if I come across this part of code, I will think there's a mistake.

Reply:
okay I will!